purism/librem/5r4/README.md

```diff
@@ -0,0 +1,133 @@
+# Purism Librem 5 revision 4
+
+Purism's [Librem 5] is a privacy-oriented Linux-friendly smartphone.
+
+[Librem 5]: https://puri.sm/products/librem-5/
+
+## Installation procedure
+
+> *Note*
+>
+> TODO: build a uuu-compatible installer.
+
+Until there's a native installer, the easiest way to install NixOS on Librem 5 seems to be using [Jumpdrive].
+
+[Jumpdrive]: https://github.com/dreemurrs-embedded/Jumpdrive
+
+### Jumpdrive
+
+Jumpdrive is a tiny Linux distribution which presents device's internal storage as USB mass storage when you connect it to a PC.
+It also provides a shell session over telnet.
+
+Follow the instructions in the repo to boot into Jumpdrive.
+Note that `uuu` is part of `nxpmicro-mfgtools` package in nixpkgs.
+
+Now, plug the device into your PC. A new block device representing Librem 5's internal MMC should appear in `/dev`.
+Note down this device path.
+
+### U-Boot
+
+> **Note**
+>
+> While upstream u-boot does support Librem 5, it can only boot using `boot.scr`, for which NixOS has no native support.
```

Comment:
not sure if useful but I'm running an u-boot that supports `/boot/extlinux/extlinux.conf`: https://forums.puri.sm/t/can-someone-with-serial-console-access-try-nixos-kernel-on-librem-5/19121/27

Reply:
Yes, that's why there's a custom u-boot packages in this repository. This note is still here from the previous PR I'm continuing and I haven't cleaned up everything yet.